arangod/StorageEngine/TransactionState.cpp

```diff
@@ -324,6 +325,7 @@ bool TransactionState::isExclusiveTransactionOnSingleCollection() const {
 }
 
 int TransactionState::checkCollectionPermission(TRI_voc_cid_t cid,
+                                                std::string const& cname,
```

Comment:
why is a name necessary if a cid is already provided?

Reply:
to get rid of the name resolution